docs/docusaurus/docs/core/define_expectations/create_an_expectation.md

```diff
@@ -0,0 +1,178 @@
+---
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import PrereqGxInstalled from '../_core_components/prerequisites/_gx_installation.md';
+
+import StandardArgumentsTable from './_expectations/_standard_arguments_table.md';
+
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+An Expectation is a verifiable assertion about your data. Expectations make implicit assumptions about your data explicit, and they provide a flexible, declarative language for describing expected behavior. They can help you better understand your data and help you improve data quality. 
+
+<h2>Prerequisites</h2>
+
+- <PrereqPythonInstalled/>.
+- <PrereqGxInstalled/>.
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Choose an Expectation to create.
+
+   GX comes with many built in Expectations to cover your data quality needs.  You can find a catalog of these Expectations in the [Expectation Gallery](https://greatexpectations.io/expectations/).  When browsing the Expectation Gallery you can filter the available Expectations by the data quality issue they address and by the Data Sources they support.  There is also a search bar that will let you filter Expectations by matching text in their name or docstring.
+
+   In your code, you will find the classes for Expectations in the `expectations` module:
+
+   ```python title="Python"
+   from great_expectations import expectations as gxe
+   ```
+
+2. Determine the Expectation's evaluation parameters
+
+   To determine the parameters your Expectation uses to evaluate data, reference the Expectation's entry in the [Expectation Gallery](https://greatexpectations.io/expectations/).  Under the **Args** section you will find a list of parameters that are necessary for the Expectation to be evaluated, along with the a description of the value that should be provided.
+
+   Parameters with a default value do not need to be set.  Parameters that are set to `None` can be provided at runtime.  Values for all other parameters must be provided when the Expectation is created.
+
+   For example, take the [**Args** section for `ExpectColumnMaxToBeBetween` in the Expectation Gallery](https://greatexpectations.io/expectations/expect_column_max_to_be_between#args).  In this section, you will see that for `ExpectColumnMaxToBeBetween`'s parameters:
+
+      - `column` must be a string.  Since it cannot be set to `None` its value must be provided when the Expectation is created.
+      - `min_value` and `max_value` can be either set when the Expectation is created, or be set to `None` when the Expectation is created and then have appropriate values passed in at runtime, instead.
+      - `strict_min` and `strict_max` both have default values that can be overriden when the Expectation is created, or left as is.  Because they cannot be set to `None` when the Expectation is created these parameters also cannot be passed in at runtime.
```

Comment:
super nit: `both have default values that can be overriden when the Expectation is created, or left as is` reads a bit weird to me. (also `overriden` is a typo). I'd think about something like `strict_min and strct_max are both optional fields; their defaults are documented with the expectation` or something. I don't know if you really "override" a default.